apps/chef_db/src/chef_sql.erl

```diff
@@ -167,13 +171,9 @@ fetch_org_metadata(OrgName) ->
 %% see also: oc_chef_authz/oc_chef_user_org_association
 %%           oc_chef_authz/oc_chef_db
 %%
-%% *************************************************************
-%%                         NOT FOR RELEASE!
-%% Temporary hack - required until org creation knows about sql associations
-%%                         NOT FOR RELEASE!
-%% *************************************************************
 -spec is_user_in_org(binary(), binary()) -> boolean() | {error, _}.
 is_user_in_org(<<"pivotal">>, _) ->
```

Comment:
Yeah this was never intended to stay - might be something we can push into oc_chef_authz, where we have something similar anyway